RELEASES.md

```diff
@@ -1,3 +1,7 @@
+# 0.8.2
+### Change
+- Issue #64 - Set blake2 to dev-dependency
+
```

Comment:
Since this will include the new bls12_381 version, the version will be 0.9

Reply:
Several nits:
1. This should be currently done under `Unreleased` not under a specific version yet
2. The format should be consistent with the guidelines related to the commit message, so this should be change into:

```
## Unreleased

### Changed
Change black2 from dependency to dev-dependency #64
````

This also should be reflected in the commit message (see for example: https://github.com/dusk-network/plonk/pull/434#pullrequestreview-606196904).

Only in the CHANGELOG, if you want to add the link to the issue, you can do:

```
Change black2 from dependency to dev-dependency [#64](https://github.com/dusk-network/jubjub/issues/64)
```

Or you can do:
```
Change black2 from dependency to dev-dependency [#64]
```

And adding the link to the footer (see: https://github.com/dusk-network/dusk-abi/blob/release-0.7/CHANGELOG.md as example).

Whatever you decided, we should be consistent within the file.